### ISSUE TITLE
Update eofs to 1.1.0

### ISSUE THREAD

**Issue by durack1** [PCMDI/pcmdi_metrics#319]
This is a duplicate of https://github.com/UV-CDAT/uvcdat/issues/1910

It should be closed once https://github.com/UV-CDAT/uvcdat/pull/1911 is merged

**Comment by durack1** [comment]
Closed by https://github.com/UV-CDAT/uvcdat/pull/1911
@gleckler1 @doutriaux1

**Comment by durack1** [comment]
Closed by https://github.com/UV-CDAT/uvcdat/pull/1911
@gleckler1 @doutriaux1
